lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -690,3 +691,11 @@ def _section_metrics(course, access):
         'post_metrics_data_csv_url': reverse('post_metrics_data_csv'),
     }
     return section_data
+
+
+def is_e_commerce_course(course_key):
```

Comment:
It's good that we have integration tests that check for the overall outcome, but we also need one or more tests that exercise this specific function.